docs/HowToFAQ/FAQIssues.md

```diff
@@ -0,0 +1,15 @@
+---
+title: How do I ask a question or file a problem report or make a suggestion about Dafny?
+---
+
+## Question
+
+How do I ask a question or file a problem report or make a suggestion about Dafny?
+
+## Answer
+
+You can ask questions about Dafny
+- on Stack Overflow, tagged as 'dafny': [https://stackoverflow.com/questions/tagged/dafny](https://stackoverflow.com/questions/tagged/dafny)
+- On the Dafny Issues log (report bugs or problems here as well): [https://github.com/dafny-lang/dafny/issues](https://github.com/dafny-lang/dafny/issues)
```

Comment:
We don't want people to ask Dafny questions in issues, I think; we have discussions for that, but even then we prefer just stackoverflow.

Reply:
Revised